pkg/kubelet/server_test.go

```diff
@@ -931,6 +936,237 @@ func TestServeExecInContainer(t *testing.T) {
 	}
 }
 
+// TODO: largely cloned from TestServeExecContainer, refactor and re-use code
+func TestServeAttachContainer(t *testing.T) {
```

Comment:
Actually, looks like git can handle the diff.  Merge is still clean after #10837 went in.